utils.py

```diff
@@ -0,0 +1,43 @@
+import math
+import random
+import sys
+sys.path.append('C://Users/jmaynard/Documents/GitHub/assignment_05')
+
```

Comment:
You should not need to do any PATH appending.  I suspect that this is why the test are failing on Travis.